schematic/store/synapse.py

```diff
@@ -2453,7 +2455,6 @@ def getDatasetProject(self, datasetId: str) -> str:
 
         # re-query if no datasets found
         if dataset_row.empty:
-            sleep(5)
```

Comment:
This should not need an extra sleep as the retry wait should handle waiting in-between retires.

Reply:
I think @GiaJordan added this here: https://github.com/Sage-Bionetworks/schematic/commit/e8b2c8e532889543f3e48a0160170322f71a5644 but it might not be needed any longer?

Reply:
Same context as the comment below

Reply:
I will leave these in for now then